src/components/CancelAppointment.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+import { useLocation, useHistory } from 'react-router-dom';
+import {FormattedDate, FormattedMessage} from 'react-intl';
+import {post} from 'api';
+import PropTypes from 'prop-types';
+
+import Body from 'components/Body';
+import Button from 'components/Button';
+import Card from 'components/Card';
+import ErrorMessage from 'components/ErrorMessage';
+import Label from 'components/Label';
+import HelpText from 'components/HelpText';
+import Input from 'components/Input';
+import {Toolbar, ToolbarList} from 'components/Toolbar';
+import VerticalSpacer from 'components/VerticalSpacer';
+
+const CancelAppointment = ({ baseUrl }) => {
+
+  const [email, setEmail] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+  const history = useHistory();
+  const queryParams = new URLSearchParams(useLocation().search);
+  const time = queryParams.get("time").split(" ")[0];
+  const submission_uuid = queryParams.get("submission_uuid");
+
+  return (
+    <Card title="Afspraak annuleren">
+        <Body component="div">
+          <p>
+            <FormattedMessage id="uwAfspraakOp" defaultMessage="U staat op het punt om uw afspraak op " />
+            {<FormattedDate
+              value={time}
+              day="numeric"
+              month="long"
+            />}
+            <FormattedMessage id="om" defaultMessage=" om " />
+            {new Date(time).toLocaleTimeString().replace(":", ".").split(":")[0]}
+            <FormattedMessage id="teAnnuleren" defaultMessage="  te annuleren. Vul ter validatie uw e-mailadres in." />
+          </p>
+
+          <VerticalSpacer pixels={60} />
```

Comment:
Not sure how we feel about a pattern like this?  

It seemed like a simple enough solution to add vertical spacing while also making a reusable solution that can be used in other places.